interface/modules/custom_modules/oe-module-weno/src/Services/TransmitProperties.php

```diff
@@ -91,9 +102,11 @@ public function createJsonObject()
      */
     public function getProviderEmail()
     {
-        $provider_info = sqlQuery("select email from users where username=? ", [$_SESSION["authUser"]]);
+        $uid = $_SESSION['authUserID'];
+        $sql = "select setting_value AS email from user_settings where setting_user = ? and setting_label = 'global:weno_provider_email'";
```

Comment:
user settings are stored in globals (they will overwrite the main global if it exists), so should be able to get above by just using $GLOBALS['weno_provider_email']